cosmoz-bottom-bar-view.js

```diff
@@ -58,6 +58,8 @@ const CosmozBottomBarView = ({
 				@computed-bar-height-changed=${ e => info.computedBarHeight = e.target.computedBarHeight } >
 				<slot></slot>
 				<slot name="extra" slot="extra"></slot>
+				<slot name="bottom-bar-toolbar" slot="bottom-bar-toolbar"></slot>
```

Comment:
Could we provide a tiny template (in Polymer too) for this to limit mistakes ?

Reply:
Good idea!